bootstrapvz/common/assets/init.d/expand-root

```diff
@@ -13,8 +13,19 @@
 prog=$(basename $0)
 logger="logger -t $prog"
 
-device_path="/dev/xvda"
+growpart=$(which growpart) || {
```

Comment:
In my other project (homeshick) I have actually gathered [a list of exit codes](https://github.com/andsens/homeshick/blob/e8d021c10c7a54fba0a81a7995ca0148c07f3a9b/lib/exit_status.sh) from different sources. It seems `1` is the only one applicable.